pkg/plugins/manager/loader/loader.go

```diff
@@ -271,7 +271,12 @@ func setDefaultNavURL(p *plugins.Plugin) {
 			p.DefaultNavURL = path.Join("/plugins/", p.ID, "/page/", include.Slug)
 		}
 		if include.Type == "dashboard" {
-			p.DefaultNavURL = path.Join("/dashboard/db/", include.Slug)
+			if include.UID == "" {
+				p.Logger().Warn("Included dashboard is missing a UID field")
```

Comment:
Falling back to `path.Join("/dashboard/db/", include.Slug)` makes no sense, right?

Reply:
Sadly not. That API is long done :rip:

Reply:
There's some related code here which does fallback to slug for whatever reason: https://github.com/grafana/grafana/blob/1a4b1184bd6fcf73d5749edb571d0e628f654078/pkg/api/index.go#L122-L129